conceptnet5/readers/emoji.py

```diff
@@ -0,0 +1,49 @@
+import xml.etree.ElementTree as et
+from conceptnet5.edges import make_edge
+from conceptnet5.formats.msgpack_stream import MsgpackStreamWriter
+from conceptnet5.uri import Licenses, concept_uri
+
+def is_sentence(text):
+	if(' ' in text and not '|' in text):
+		return True
+	else:
+		return False
+
+def amt_words(text):
+	return text.count('|')+1
+
+def strip_words(text):
+	res =  text.split('|')
+	for i in range(len(res)):
+		res[i] = res[i].replace(" ","")
+	return res
+
+def handle_file(input_file, output_file):
+	tree = et.parse(input_file)
+	root = tree.getroot()
+	lang = root[0][1].attrib['type']
+	out = MsgpackStreamWriter(output_file)
+	for annotation in root[1]:
+		if(not is_sentence(annotation.text)):
+			rel = '/r/SymbolOf'
+			start = '/c/mul/'+annotation.attrib['cp']
```

Comment:
Except I actually think you want to be using `standardized_concept_uri`, particularly for the natural language side, because that makes sure it's in the right format with spaces converted to underscores.